reorgdetector/config.go

```diff
@@ -17,6 +18,11 @@ type Config struct {
 
 	// CheckReorgsInterval is the interval to check for reorgs in tracked blocks
 	CheckReorgsInterval types.Duration `mapstructure:"CheckReorgsInterval"`
+	// FinalizedBlockType indicates the status of the blocks that will be queried in order to sync
+	// if finalizedBlock == "LatestBlock" then is like to disable reorg detector
+	// there are no chances to reorg
+	FinalizedBlock etherman.BlockNumberFinality `jsonschema:"enum=LatestBlock, enum=SafeBlock, enum=PendingBlock, enum=FinalizedBlock, enum=EarliestBlock" mapstructure:"FinalizedBlock"` //nolint:lll
```

Comment:
Maybe it is like this to be consistent with other components, but I would remove the "block" from all the options: 
```suggestion
	FinalizedBlock etherman.BlockNumberFinality `jsonschema:"enum=Latest, enum=Safe, enum=Pending, enum=Finalized, enum=Earliest" mapstructure:"Finalized"` //nolint:lll
```

Reply:
I agree, but  the rest of components use this name to set the block finality